### ISSUE TITLE
Dun Niffelem stuck in quest stage after quests complete

### ISSUE THREAD

**Issue by Zraebiel** [dalaranwow/dalaran-wow#4461]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. I flew to the Dun Niffelem Flight path and am stuck in a quest phase
2. I've already completed the quest chain and am revered with Hodir
![dunniffelembug_021917_001](https://cloud.githubusercontent.com/assets/24212815/23104696/b823895c-f687-11e6-9904-7d0d117f97d3.png)

3. I cannot see the quartermaster or buy from him.  This means I cannot buy another shoulder enchantment which hurts this characters progression.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Zraebiel** [comment]
Here are a couple more screen shots and can submit more if needed.

![dunniffelembug_021917_002](https://cloud.githubusercontent.com/assets/24212815/23104873/c82253e4-f68a-11e6-867f-25bac3defb48.png)

^Shows I've bought the enchant from the quartermaster before^

![dunniffelembug_021917_003](https://cloud.githubusercontent.com/assets/24212815/23104877/df5b09fc-f68a-11e6-89c5-a7130c2bae45.png)

So please help and thank you!

**Comment by Rushor** [comment]
You are in the wrong phase then for some reason. Please contact a gamemaster and describe your issue there. He will help you as fast as he can and puts you in the correct phase.

Thanks.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)

**Comment by Zraebiel** [comment]
I already have a ticket in so do I need to abandon that first?

**Comment by Rushor** [comment]
no if you already wrote a ticket, a gamemaster will review it once he got time :)

**Comment by Zraebiel** [comment]
This is still an issue and should not be closed.

**Comment by Rushor** [comment]
individual issue, it does not affect all player as far as i know. do you already talked to a gamemaster?

**Comment by Zraebiel** [comment]
No I have had the ticket up since before posting here with no answer.  At this point I may be forced to lvl another character and do the quest chain with them to buy the boa enchant.

**Comment by Rushor** [comment]
i would suggest you to open up a new ticket then for the gamemaster

**Comment by Zraebiel** [comment]
Ok I put up another ticket.  I also noticed someone else in my same phase and we talked a moment.  He was having the same problem.  So there is at least 1 more and im sure even more have the problem and dont realize it yet.

**Comment by Zraebiel** [comment]
Doesn't seem like they care.  I literally have to lvl up another character just to do the quest chain again to buy from the quartermaster...lol.  So sad...

**Comment by Teridan-dal-wow** [comment]
Hello Zraebiel,
Here are some "things" you can try to do to solve your issue:
 - Fly away from Dun Niffelem (far away) > relog > come back and check,
 - Go to Dun Niffelem and que for rdf/rbg > accept the invitation > complete the rbg/rdf > check,
 - Go to K3 base > que for rdf/rbg > wait for pop up massage to come (don't accept it) > once you have it take the flightpath towards Dun Niffelem > when you are near the base (before you see the icewalls of the town (Dun Niffelem)) accept the invitation > complete the rdf/rbg > you should be back on your flightpath (almost at the base) > check,
 - Wait for GM to help you with your ticket (be online on that character as much as possilbe).

On a sidenote, can tell us how did you manage to get yourself in that situation? If you can provide the steps you did and someone from the staff can reproduce your problem this issue can be valid and reopened. 
Thanks and goodluck.

**Comment by Zraebiel** [comment]
Ok Ill try those rdf suggestions those sound interesting.  I already did the log out far away login/check thing of course and I've been waiting on a gm with a ticket for about a week now.  All I did was take the flight path out there I can't think of anything special that occurred.  I believe around the same time I was unable to mount my flying mount at Krassus Landing.  That all I can remember.

**Comment by Zraebiel** [comment]
I tried those rdf suggestions and they didnt work.  I saw someone there doing the quest though so that confirms for me it is a quest phase.  That person jumped on one of the blue proto-drakes that are chained at the top of the area.  Is it not possible to reset the quest chain?  Any other suggestions in case a gm never responds?  I really appreciate the suggestions here so far.
